### ISSUE TITLE
Python 3.10: TypeError: shield() got an unexpected keyword argument 'loop'

### ISSUE THREAD

**Issue by jwhitlock** [sparkmeter/sentry2csv#18]
When running with Python 3.10, I get this traceback:

```
Fetching issues page 1
/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/aiohttp/connector.py:964: RuntimeWarning: coroutine 'TCPConnector._resolve_host' was never awaited
  hosts = await asyncio.shield(self._resolve_host(
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
Traceback (most recent call last):
  File "/Users/john/.local/bin/sentry2csv", line 8, in <module>
    sys.exit(main())
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/sentry2csv/sentry2csv.py", line 190, in main
    loop.run_until_complete(
  File "/opt/homebrew/Cellar/python@3.10/3.10.2/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/base_events.py", line 641, in run_until_complete
    return future.result()
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/sentry2csv/sentry2csv.py", line 139, in export
    issues = await fetch_issues(session, issues_url)
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/sentry2csv/sentry2csv.py", line 76, in fetch_issues
    resp, links = await fetch(
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/sentry2csv/sentry2csv.py", line 47, in fetch
    async with session.get(url, params=params) as response:
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/aiohttp/client.py", line 1012, in __aenter__
    self._resp = await self._coro
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/aiohttp/client.py", line 480, in _request
    conn = await self._connector.connect(
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/aiohttp/connector.py", line 523, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/aiohttp/connector.py", line 858, in _create_connection
    _, proto = await self._create_direct_connection(
  File "/Users/john/.local/pipx/venvs/sentry2csv/lib/python3.10/site-packages/aiohttp/connector.py", line 964, in _create_direct_connection
    hosts = await asyncio.shield(self._resolve_host(
TypeError: shield() got an unexpected keyword argument 'loop'
```

It works under Python 3.9.

**Comment by arusahni** [comment]
Thanks for the detailed report! Looks like I'll need to upgrade a dependency. I'll investigate and most likely have a release ready to go by next week.

**Comment by arusahni** [comment]
I have a fix merged in on `master`. I'll cut a new release in the next few days.

**Comment by arusahni** [comment]
This has been released as v1.2.1, and deployed to pypi. Thanks for the report, @jwhitlock!
